environments.ini

```diff
@@ -1483,6 +1483,23 @@ build_flags =
   '-DGateway_Name="OpenMQTTGateway_ESP32_BLE"'
 custom_description = BLE gateway on the C3
 
+[env:airm2m_core_esp32c3]
+platform = ${com.esp32_c3_s3_platform}
+board = airm2m_core_esp32c3
+board_build.partitions = min_spiffs.csv
+lib_deps =
+  ${com-esp.lib_deps}
+  ${libraries.wifimanager32}
+  ${libraries.ble}
+  ${libraries.decoder}
+build_flags =
+  ${com-esp.build_flags}
+  '-DZgatewayBT="BT"'
+  '-DLED_INFO=19'
+  '-DLED_SEND_RECEIVE=20'
+  '-DNO_INT_TEMP_READING=true' ; Internal temperature reading not building on ESP32 C3 or S3
+  '-DGateway_Name="OpenMQTTGateway_AirM2M_Core_ESP32C3"'
+
```

Comment:
Could you add a `custom_description` please

Reply:
I added it

Reply:
thanks